src/senaite/core/content/senaitesetup.py

```diff
@@ -114,3 +144,31 @@ def setEnableGlobalAuditlog(self, value):
             catalog.manage_catalogClear()
         mutator = self.mutator("enable_global_auditlog")
         return mutator(self, value)
+
+    @security.protected(permissions.View)
+    def getSiteLogo(self):
+        """Returns the global site logo
+        """
+        accessor = self.accessor("site_logo")
+        return accessor(self)
+
+    @security.protected(permissions.ModifyPortalContent)
+    def setSiteLogo(self, value):
+        """Set the site logo
+        """
+        mutator = self.mutator("site_logo")
+        return mutator(self, value)
+
+    @security.protected(permissions.View)
+    def getSiteLogoCSS(self):
+        """Returns the global site logo
+        """
+        accessor = self.accessor("site_logo_css")
+        return accessor(self)
+
+    @security.protected(permissions.ModifyPortalContent)
+    def setSiteLogoCSS(self, value):
+        """Set the site logo
+        """
+        mutator = self.mutator("site_logo_css")
+        return mutator(self, value)
```

Comment:
The line break at end of file that got lost here will be re-added in https://github.com/senaite/senaite.core/pull/2078#discussion_r934369884